src/main/MainScreenContainer.js

```diff
@@ -12,14 +12,14 @@ class MainScreenContainer extends React.Component {
   fetchOlder = () => {
     const { auth, fetching, caughtUp, anchor, narrow, fetchMessages } = this.props;
     if (!fetching.older && !caughtUp.older && anchor) {
-      fetchMessages(auth, anchor.older, 20, 0, narrow);
+      fetchMessages(auth, anchor.older, 50, 0, narrow);
```

Comment:
I'd be nice if we could factor out this constant somewhere so it's easy to change while testing